tests/integration/test_contract.py

```diff
@@ -116,6 +118,10 @@ def test_deployed_contract(self):
         assert result["exists"]
         assert result["value"] == value
 
+        # Upgrade contract
+        tx_res = deployed_contract.upgrade({}, wallet, CONTRACT_PATH)
+        assert tx_res.response
```

Comment:
It's not obvious to me that this means the upgrade was successful. Maybe we can check something like that the `code_id` changed and the `address` stayed the same?